src/main/tray.ts

```diff
@@ -233,13 +239,21 @@ export class Tray {
       logo = this.trayIconSet.error;
     }
 
-    const stateMenu = this.contextMenuItems.find(item => item.id === 'state');
+    const [stateMenu, containerRuntimeMenu] = this.contextMenuItems
```

Comment:
Strictly speaking, this is wrong, because it depends on the ordering of `this.contextMenuItems`. While we can verify the order is correct, we don't really gain anything by assigning the two values with one filter call. Might as well split them out to
```
    const stateMenu = this.contextMenuItems.find(item => item.id === 'state');
    const containerRuntimeMenu = this.contextMenuItems.find(item => item.id === 'container-runtime');
```

Reply:
Good point, changed to make use of `filter`